bitbots_vision/config/visionparams.yaml

```diff
@@ -22,7 +22,8 @@ ROS_field_mask_image_msg_topic: 'field_mask'  # ROS topic of the field mask debu
 ROS_dynamic_color_space_field_mask_image_msg_topic: 'dynamic_field_mask'  # ROS topic of the dynamic color space field mask debug image message
 
 fcnn_model_path: '2019_06_timon_basler'  # Name of the ball fcnn model
-yolo_model_path: '2019_07_03_jonas_yolo'  # Name of the yolo model
+yolo_darknet_model_path: '2019_07_03_jonas_yolo'  # Name of the yolo model
+yolo_openvino_model_path: 'test'  # Name of the yolo model
```

Comment:
No, because no OpenVINO models are deployed on the Server and this is the name of my test model. But we should indeed deploy the converted models and find a good name for them.